test/acceptance/features/bindKnativeService.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Bind an application to a knative service
+
+  As a user of Service Binding Operator
+  I want to bind applications to knative services it depends on
```

Comment:
single app or multiple apps? also if it is single app then we might want to rename the file to AppBindKnativeService.feature. Please refer https://github.com/redhat-developer/service-binding-operator/pull/598#issuecomment-669760403

Reply:
For not it is to bind single app, renamed to bindAppToKnativeService.feature
Let me know if this is fine.

Reply:
@shvasude The name `bindAppToKnativeService.feature` suggest we're binding an app to knative service, but we're not doing that. We're binding an app to backend service where the app is regulated or controlled by knative-service. In this example, we're not directly making any changes to the application deployment but we're updating the knative-service which further takes care of application. 
So @Avni-Sharma IMHO `bindKnativeService.feature` is good enough even if we do bind multiple knative services they will be added under the same feature with a different scenario name.

Reply:
agree with @pratikjagrut here. but then the feature title should be updated:

```
Feature: Bind Knative service to a service

As a user of Service Binding Operator
I want to bind Knative service to services it depends on
```

Reply:
Done, feature file name: bindKnativeService.feature
and 
Feature: Bind knative service to a service
  As a user of Service Binding Operator
  I want to bind knative service to services it depends on

Reply:
or should we say ?
As a user of Service Binding Operator
I want to bind app running as knative service to services it depends on

Reply:
@pedjak Shouldn't the feature title be like:
```
Feature: Bind Knative service to a backend service


Feature: Bind Knative service to a database
```
Shouldn't we be articulate about which service is binding to which service?

Reply:
As far as i remember from an initial discussions is that the service can be any service within a feature file that will be part of different scenario. for now, i am sticking to this
https://github.com/redhat-developer/service-binding-operator/pull/567#discussion_r467027706